coalib/results/TextPosition.py

```diff
@@ -47,12 +101,15 @@ def __le__(self, other):
         """
         if self.line is None or other.line is None:
             return True
+
         if self.line == other.line:
-            return (True
-                    if self.column is None or
-                    other.column is None
-                    else
-                    self.column <= other.column)
+            if self.column is None or other.column is None:
+                return True
+            else:
+                if self.column <= other.column:
+                    return True
+                else:
+                    return False
```

Comment:
This is what I had before, then I changed it as adtac asked to use simple if else. (as it was confusing)